src/main/java/ezschedule/model/event/EventMatchesDatePredicate.java

```diff
@@ -0,0 +1,20 @@
+package ezschedule.model.event;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Event}'s {@code Date} matches the date given.
+ */
+public class EventMatchesDatePredicate implements Predicate<Event> {
+
+    private final Date date;
+
+    public EventMatchesDatePredicate(Date date) {
+        this.date = date;
+    }
+
+    @Override
+    public boolean test(Event event) {
```

Comment:
Minor issue: method name does not conform to code quality

Reply:
I followed the previous Predicate class, should be an easy fix for v1.4